wled00/bus_wrapper.h

```diff
@@ -245,7 +245,10 @@
 #endif
 
 // RMT driver selection
-#if !defined(WLED_USE_SHARED_RMT)  && !defined(__riscv)
+#if defined(CONFIG_IDF_TARGET_ESP32C6)
```

Comment:
@willmmiles this is a kind of preprocessor hack, that actually hides " NeoEsp32BitBang" under the  NeoEsp32RmtMethod label. Not sure if we should do it this way, what do you think?